src/ProjectService.Business/Commands/File/RemoveFileCommand.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using LT.DigitalOffice.Kernel.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Kernel.Constants;
+using LT.DigitalOffice.Kernel.Enums;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Models.Broker.Requests.File;
+using LT.DigitalOffice.ProjectService.Business.Commands.File.Interfaces;
+using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
+using MassTransit;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+
+namespace LT.DigitalOffice.ProjectService.Business.Commands.File
+{
+  public class RemoveFileCommand : IRemoveFileCommand
+  {
+    private readonly IFileRepository _repository;
+    private readonly IRequestClient<IRemoveFilesRequest> _rcFiles;
+    private readonly ILogger<RemoveFileCommand> _logger;
+    private readonly IAccessValidator _accessValidator;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IUserRepository _userRepository;
+
+    private async Task<bool> RemoveFileAsync(List<Guid> ids, List<string> errors)
+    {
+      if (ids == null || !ids.Any())
+      {
+        return false;
+      }
+
+      string logMessage = "Errors while removing files ids {ids}. Errors: {Errors}";
+
+      try
+      {
+        Response<IOperationResult<bool>> response =
+          await _rcFiles.GetResponse<IOperationResult<bool>>(
+            IRemoveFilesRequest.CreateObj(ids));
+
+        if (response.Message.IsSuccess)
+        {
+          return true;
```

Comment:
return response.Message.Body, it can be false

Reply:
done